<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки подключения</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="HostLayout" stretch="1,3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Хост:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="HostEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PortLayout" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Порт:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PortEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="DBLayout" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Имя БД:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DBEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="UserLayout" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Имя пользователя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UserEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PasswordLayout" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Пароль:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="SaveButton">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
